test/unit/runnable.spec.js

```diff
@@ -702,6 +702,7 @@ describe('Runnable(title, fn)', function () {
       runnable.timeout(10);
       runnable.resetTimeout();
       runnable.timeout(0);
+      runnable.run();
```

Comment:
The test was stopping Node from exiting because the `runnable`'s `.timer` was still pending.